saw-remote-api/src/SAWServer/LLVMCrucibleSetup.hs

```diff
@@ -108,7 +123,7 @@ interpretLLVMSetup ::
   [SetupStep LLVM.Type] ->
   LLVMCrucibleSetupM ()
 interpretLLVMSetup fileReader bic cenv0 ss =
-  runStateT (traverse_ go ss) (mempty, cenv0) *> pure ()
+  runStateT (traverse_ go ss) (mempty, cenv0) $> ()
```

Comment:
If you use `evalStateT` instead of `runStateT` here, you won't need the `$> ()` part here, I believe.

Reply:
Changed, thanks!